dev/how-to/scale/horizontally.md

```diff
@@ -13,7 +13,7 @@ TiDB 集群可以在不影响线上服务的情况下动态进行扩容和缩容
 
 下面分别介绍如何增加或者删除 PD，TiKV 以及 TiDB 的节点。
 
-下面用到的 pd-ctl 文档可以参考 [pd-control](/tools/pd-control.md)。
+下面用到的 pd-ctl 文档可以参考 [pd-control](/reference/tools/pd-control.md)。
```

Comment:
Please also update this link in `v3.0` and `v2.1`.

Reply:
@lilin90 This dead link only exists in `dev`.

Reply:
@CaitinChen I just checked. They do exist in `v3.0` and `v2.1`. Take another look?

Reply:
@lilin90 Thanks for your check! Updated~